xml/deploy_cephadm.xml

```diff
@@ -51,50 +136,508 @@
      &sminion;s need to correctly resolve the &smaster;'s host name over the
      network. By default, they look for the <systemitem>salt</systemitem> host
      name, but you can specify any other network-reachable host name in the
-     <filename>/etc/salt/minion</filename> file, see
-     DEAD LINK.
+     <filename>/etc/salt/minion</filename> file.
     </para>
    </listitem>
   </itemizedlist>
- </sect1>
- <sect1 xml:id="deploy-cephadm-admnode">
-  <title>Configure the &adm;</title>
 
-  <para></para>
+  <procedure>
+   <step>
+    <para>
+     Install the <literal>salt-master</literal> and
+     <literal>salt-minion</literal> packages on the &smaster; node:
+    </para>
+<screen>&prompt.smaster;zypper in salt-master salt-minion</screen>
+    <para>
+     Check that the <systemitem>salt-master</systemitem> service is enabled and
+     started, and enable and start it if needed:
+    </para>
+<screen>&prompt.smaster;systemctl enable salt-master.service
+&prompt.smaster;systemctl start salt-master.service</screen>
+   </step>
+   <step>
+    <para>
+     If you intend to use firewall, verify that the &smaster; node has ports
+     4505 and 4506 open to all &sminion; nodes. If the ports are closed, you
+     can open them using the <command>yast2 firewall</command> command by
+     allowing the <guimenu>SaltStack</guimenu> service.
+    </para>
+   </step>
+   <step>
+    <para>
+     Install the package <literal>salt-minion</literal> on all minion nodes.
+    </para>
+<screen>&prompt.sminion;zypper in salt-minion</screen>
+    <para>
+     Make sure that the <emphasis>fully qualified domain name</emphasis> of
+     each node can be resolved to the public network IP address by all other
```

Comment:
```suggestion
     each node can be resolved to an IP address on the public network by all the other
```

Reply:
If I understand this correctly -- the FQDN should be on the public network, yes? Not any other, e.g. an inter-node network?